### PR TITLE
chore: CRW-4828 updating for 3.9 release 

### PR DESCRIPTION
**Pull request by SDawley**
Adding 3.10 as 3.x 
Removing 3.7

Onwards towards 3.9
https://issues.redhat.com/browse/CRW-4828

### CODE REVIEW

File: dependencies/job-config.json [hunk 1]
```diff
@@ -441,172 +440,174 @@
...
+        "CSV_VERSION": "3.10.0",
+        "CSV_VERSION_PREV": "3.8.0"
       }
     }
   },
   "Other": {
     "Errata": {
-      "3.7": "118664",
       "3.8": "116894",
       "3.9": "n/a",
```
Comment (nickboldt) [inline dependencies/job-config.json:536]: could add the new errata here or do it in another PR

File: dependencies/job-config.json [hunk 2]
```diff
@@ -441,172 +440,174 @@
...
+      "3.10": "n/a",
       "3.x": "n/a"
     },
     "Epic": {
-      "3.7": "CRW-4792",
       "3.8": "CRW-4586",
       "3.9": "n/a",
```
Comment (nickboldt) [inline dependencies/job-config.json:542]: could add the epic for 3.9 here or in a followup PR

File: dependencies/job-config.json [hunk 3]
```diff
@@ -441,172 +440,174 @@
...
       "3.8": "7.72.0",
       "3.9": "next",
+      "3.10": "next",
       "3.x": "next"
     },
     "THEIA_BUILD_PARAMS": {},
     "OPENSHIFT_VERSIONS_SUPPORTED": {
-      "3.7": [
+      "3.8": [
+        "v4.14",
```
Comment (nickboldt) [inline dependencies/job-config.json:597]: well, technically we don't support 4.14 since it's not live yet (planned for Oct). Can't recall how this field is used ... for the list of IIBs to list in emails and copy to quay, maybe? if so this is fine.

File: dependencies/job-config.json [hunk 4]
```diff
@@ -441,172 +440,174 @@
...
         "v4.11",
         "v4.10"
       ],
-      "3.8": [
+      "3.9": [
         "v4.14",
         "v4.13",
         "v4.12",
         "v4.11",
         "v4.10"
```
Comment (nickboldt) [inline dependencies/job-config.json:608]: I think we can drop v4.10 since it's EOL in a week. Might have to update the dockefile to set a new minimum OCP version too.

**Review by nickboldt — APPROVED** [review]
LGTM, just some followups

File: dependencies/job-config.json (hunk 3 above)
Comment (SDawley) [reply dependencies/job-config.json:597]: Push-latest-container-to-quay, copyIIBsToQuay and email, yeah.

File: dependencies/job-config.json (hunk 4 above)
Comment (SDawley) [reply dependencies/job-config.json]: OK, I've made a note to update the Dockerfiles, but that will probably happen next week.
